modules/videopress/class.videopress-gutenberg.php

```diff
@@ -6,8 +6,16 @@ class VideoPress_Gutenberg {
 	 * Initialize the VideoPress Gutenberg extension
 	 */
 	public static function init() {
-		// Should not initialize if Gutenberg is not available
-		if ( function_exists( 'register_block_type' ) ) {
+		// Should not initialize if Gutenberg is not available or if Jetpack is not active
+		if (
+			( function_exists( 'register_block_type' ) )
+			&&
+			(
+				( method_exists( 'Jetpack', 'is_active' ) && Jetpack::is_active() )
+				||
+				( defined( 'IS_WPCOM' ) && IS_WPCOM )
```

Comment:
Since we need the `is_active` check anyway, I personally don't mind the current implementation. It is clear enough to me, and having it all in there makes it easier to keep the feature in sync.

Reply:
Good idea @rodrigoi! 

My only concern about is that we would end up maintaining code in 2 places which is what we want to avoid (and the reason why Fusion exists or why we want to improve our dev workflow: pafL3P-gQ-p2).

Especially if the condition is not very common, because we will need to add a new filter for each new feature. In this case for example, we need to run a piece of code always on WP.com and only if Jetpack is active on Jetpack sites, but a quick search on the codebase revealed it is not very usual. 

Also, this will make a bit more complex the review process:
- Each feature shared between Jetpack and WP.com will define a new filter whenever we need to be agnostic of the environment. This will create a PR in Jetpack and a diff in WP.com
- Since Fusion syncs the previous diff with the PR, we will need to open a new diff for implementing the filter in WP.com.